packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/index.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { PureComponent } from 'react'
 import { FormattedMessage } from 'react-intl'
 import { connect, ConnectedProps } from 'react-redux'
+import { isEqual } from 'lodash'
```

Comment:
we don't have the lodash module installed on wallet. We use a library called Ramda that has a similar method though https://ramdajs.com/docs/#equals

Reply:
ok, will change that to use ramdajs.